can-rest-model.md

```diff
@@ -247,32 +240,49 @@ with their own name and complete status:
 ```
 
 It can be nice to have the individual subtasks have a reference to their parent `todo`. For example, this
-makes updating the subtask easier.  The following makes it so calling a `subtask`'s `.save()` actually
-calls it's `todo`'s `.save()` method:
+makes updating the subtask easier.  The following makes it so calling a `subtask`’s `.save()` actually
+calls its `todo`’s `.save()` method:
 
 ```js
-import {DefineMap, DefineList, restModel} from "//unpkg.com/can@5/core.mjs";
-import {todoFixture} from "//unpkg.com/can-demo-models@5";
+import { ObservableArray, ObservableObject, restModel, type } from "//unpkg.com/can@5/core.mjs";
+import { todoFixture } from "//unpkg.com/can-demo-models@6";
 
 // Model subtask
-const Subtask = DefineMap.extend("Subtask",{
-    name: "string",
-    complete: "boolean",
-    // parentTodo should not be serialized
-    parentTodo: {serialize: false, type: "any"},
+class Subtask extends ObservableObject {
+    static props = {
+        name: String,
+        complete: Boolean,
+
+        // parentTodo should not be serialized
+        parentTodo: {serialize: false, type: type.Any}
+    };
+
     // a save utility that actually saves the parent todo
-    save(){
+    save() {
         this.parentTodo.save();
     }
-});
+}
 
 // Model a list of subtasks to add the `parentTodo` to all subtasks
-Subtask.List = DefineList.extend("Subtasks",{
+class Subtasks extends ObservableArray {
+    static props = {
+        // If parentTodo is set, update all the subtasks' parentTodo
+        parentTodo: {
+            set(parentTodo) {
+                this.forEach((subtask) => {
+                    subtask.parentTodo = parentTodo;
+                });
+                return parentTodo;
+            },
+            serialize: false
+        }
+    };
+
     // Defines the items in the subtasks list
-    "#": {
-        Type: Subtask,
+    static items = {
+        type: Subtask,
         // If subtasks are added, set their parentTodo
-        added(subtasks){
+        added(subtasks) {
```

Comment:
The `added` event is not supported by `ObservableArray` so we'll need to figure out how to update these docs.